cocos_palm.lua

```diff
@@ -82,8 +82,8 @@ minetest.register_abm({
 	nodenames = { "moretrees:palm_fruit_trunk_gen" },
 	interval = 6,
 	chance = 1,
-	min_y = biome_lib.mapgen_elevation_limit.min,
-	max_y = biome_lib.mapgen_elevation_limit.max,
+	min_y = -16,
+	max_y = 48,
```

Comment:
+1 to +32 should be enough

Reply:
values taken straight from biome lib. this is invalid/scope creep. file and issue if you would like it changed